src/styles/main.scss

```
@@ -1,3 +1,70 @@
+$day-size: 100px;
+$gap: 1px;
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.celendar {
+  display: flex;
+  margin: auto;
+  width: 707px;
+  font-family: Arial, Helvetica, sans-serif;
```

Use flex with 1px gap and limit its width to exactly 7 columns + 10px paddings
```suggestion
.celendar {
  display: flex;
  margin: auto;
  padding: 10px;
  width: 707px;
  font-family: Arial, Helvetica, sans-serif;
```